frontend/public/components/monitoring/receiver-forms/pagerduty-receiver-form.tsx

```diff
@@ -2,90 +2,165 @@
 import * as _ from 'lodash-es';
 import * as React from 'react';
 
-import { SectionHeading } from '../../utils';
 import { RadioInput } from '../../radio';
+import { SaveAsDefaultCheckbox } from './alert-manager-receiver-forms';
 
-export const Form = ({ formValues, dispatchFormChange }) => (
-  <div data-test-id="pagerduty-receiver-form" className="co-m-pane__body--section-heading">
-    <SectionHeading text="PagerDuty Configuration" />
-    <div className="form-group">
-      <label className="control-label">Integration Type</label>
-      <div>
-        <RadioInput
-          title="Events API v2"
-          name="pagerDutyIntegrationType"
-          id="integration-type-events"
-          value="events"
-          onChange={(e) =>
-            dispatchFormChange({
-              type: 'setFormValues',
-              payload: { pagerDutyIntegrationType: e.target.value },
-            })
-          }
-          checked={formValues.pagerDutyIntegrationType === 'events'}
-          inline
-        />
-        <RadioInput
-          title="Prometheus"
-          name="pagerDutyIntegrationType"
-          data-test-id="integration-type-prometheus"
-          value="prometheus"
+const GLOBAL_FIELDS = ['url']; //TODO follow up PR will add advanced fields
+
+// Ex: converts 'client_url' to 'pagerdutyClientUrl'
```

Comment:
I'd really like to avoid doing converting names like this if we can

Reply:
Tracking in another comment